Marlin/pins_VOXEL8_GEN3C.h

```diff
@@ -144,8 +144,9 @@ DIGITAL POTENTIOMETER PINS
 /*************************
 	  ADMINISTRATIVE PINS
 *************************/
-#define V_MONITOR_PIN           79  //! Monitor voltage to print engine
-#define PS_ON_PIN               4
+#define PS_MONITOR_PIN           8   // A8 Monitor voltage to print engine
+#define PS_FORCE_ON_LL          70  // Enable 24V to system - Logic Low
+#define PS_ON_PIN               -1
```

Comment:
PS_ON_PIN is part of Marlin that we don't use. I just defined it as -1, because for some reason it was defined as 4 previously, which is Bed Heat.